force-app/main/default/lwc/programPlanSettings/programPlanSettings.js

```diff
@@ -0,0 +1,95 @@
+import { LightningElement, track, wire, api } from "lwc";
+import { refreshApex } from "@salesforce/apex";
+
+import getProgramPlanSettingsVModel from "@salesforce/apex/ProgramPlanSettingsController.getProgramPlanSettingsVModel";
+
+//Program Plan Settings Labels
+import stgProgramPlanSettingsNav from "@salesforce/label/c.stgProgramPlanSettingsNav";
+import stgProgramPlanSettingsTitle from "@salesforce/label/c.stgProgramPlanSettingsTitle";
+import stgNestedPlanRequirementPP from "@salesforce/label/c.stgNestedPlanRequirementPP";
+import stgHelpNestedPlanRequirementPP from "@salesforce/label/c.stgHelpNestedPlanRequirementPP";
+
+export default class ProgramPlanSettings extends LightningElement {
+    isEditMode = false;
+    affordancesDisabledToggle = false;
+
+    @track programPlanSettingsVModel;
+    @track programPlanSettingsVModelWireResult;
+
+    inputAttributeReference = {
+        validateProgramPlanToggleId: "validateProgramPlan",
+    };
+
+    labelReference = {
+        programPlanSettingsPageTitle: stgProgramPlanSettingsNav,
+        programPlanSettingsHeading: stgProgramPlanSettingsTitle,
```

Comment:
super minor but the label reference here is very confusing, we are using nav for the title and title for the heading? Maybe we can just use the original label name for now.

Reply:
I renamed references to be less confusing